src/java.base/share/classes/jdk/internal/util/ReferencedKeyMap.java

```diff
@@ -338,34 +339,91 @@ public void removeStaleReferences() {
 
     /**
      * Puts an entry where the key and the value are the same. Used for
-     * interning values.
+     * interning values in a set.
      *
      * @implNote Requires a {@link ReferencedKeyMap} whose {@code V} type
```

Comment:
I think it still serves a warning to not use this form of the `intern` method during phase1. I will add a note that they should use an UnaryOperator instance instead of a lambda.